hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/contracts/services/AzureServiceErrorCode.java

```diff
@@ -99,24 +104,22 @@ public static AzureServiceErrorCode getAzureServiceCode(int httpStatusCode, Stri
         return azureServiceErrorCode;
       }
     }
-
     return UNKNOWN;
   }
 
-  public static AzureServiceErrorCode getAzureServiceCode(int httpStatusCode, String errorCode, final String errorMessage) {
+  public static AzureServiceErrorCode getAzureServiceCode(int httpStatusCode, String errorCode, String errorMessage) {
```

Comment:
Why was final removed from errorMessageString??
Also why do we need to split the error message?